src/vcBPA.cpp

```diff
@@ -97,33 +97,39 @@ struct vcBPAGrid
   }
 };
 
+struct vcBPAOctNode
+{
+  udDouble3 center;
+  udDouble3 extents;
+  bool visited;
+
+  static vcBPAOctNode create(udDouble3 center, udDouble3 extents)
```

Comment:
ProperCase for function names

Reply:
This is consistent with the rest of the file and udMath, which is where I got the `create` idea from. To my knowledge, this is one of the only places we use this pattern in this repo too, however, I'm willing to change it in a future PR if you want?